packages/flutter_tools/test/general.shard/ios/xcodeproj_test.dart

```diff
@@ -48,6 +48,7 @@ void main() {
       platform: platform,
       processManager: processManager,
       terminal: terminal,
+      usage: null,
```

Comment:
In the future we'll likely want to make these non-nullable. Consider creating a `FlutterUsage.test()` which can be created at low cost to avoid explicitly passing null